app/workers/subject_workflow_status_count_worker.rb

```diff
@@ -0,0 +1,21 @@
+class SubjectWorkflowStatusCountWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :really_high
+
+  sidekiq_options congestion: {
+    interval: ENV.fetch("sws_count_worker_interval", 30),
```

Comment:
happy to modify the defaults here, thought this was sensible (for random dist on selection) and added ability to override with ENV vars.